src/util/schema.tsx

```diff
@@ -109,7 +111,7 @@ export async function syncConnectionSchema(options?: SchemaSyncOptions) {
 		for (const [idx, tableName] of tablesToSync.entries()) {
 			adapter.log("Schema", `Updating table ${tableName}`);
 
-			const tableStruct = tbInfoMap[idx];
+			const tableStruct = tbInfoMap[idx].result as SchemaInfoTB;
```

Comment:
Hmm, while it is not my code base, wouldn't it be possible to add a generic to the executeQuery, which then can be passed to the under-the-hood `query_raw`? That way, casting would not be necessary.